samza-core/src/main/scala/org/apache/samza/container/SamzaContainer.scala

```diff
@@ -106,6 +107,11 @@ object SamzaContainer extends Logging {
     putMDC("jobName", config.getName.getOrElse(throw new SamzaException("can not find the job name")))
     putMDC("jobId", config.getJobId.getOrElse("1"))
     var jmxServer: JmxServer = null
+    val appRunner = if (config.containsKey(StreamApplication.APP_CLASS_CONFIG)) {
+      ApplicationRunner.fromConfig(config)
+    } else {
+      null
```

Comment:
Will this work for standalone? Will they have the app class specified in the config?